science/libsbmlsim/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        libsbmlsim libsbmlsim 1.4.0 v
+name                libsbmlsim
+categories          science
+platforms           darwin
+maintainers         {@funasoul gmail.com:funasoul} openmaintainer
+license             LGPL-2.1+
+
+cmake.out_of_source yes
```

Comment:
I believe that this line is not needed since you are using the cmake 1.1 portgroup. 

In general, this portfile looks great!

Reply:
Thanks, I've updated the Portfile. Please check my commit if you have a time.